tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -589,6 +595,18 @@ const pauseResumeUpload = (page: Page): Promise<void> => {
   return page.locator(pauseResumeUploadButton).click()
 }
 
+export const navigatePreviousMedia = ({ page }): Promise<void> => {
+  return page.locator(navigatePreviousButton).click()
+}
```

Comment:
```suggestion
export const navigatePreviousMedia = async (page): Promise<void> => {
  await Promise.all(
    page.waitForResponse(
        (resp) =>
          resp.status() === 207 &&
          resp.request().method() === 'PROPFIND' &&
          resp.url().endsWith("/v")
      ),
    page.locator(navigatePreviousButton).click()
  )
}
```

Reply:
was this not working?
```js
page.waitForResponse(
        (resp) =>
        resp.status() === 207 &&
        resp.request().method() === 'PROPFIND' &&
        resp.url().endsWith("/v")
    )
```

Reply:
no, this doesn't work.

Reply:
might be different in different environments